lib/src/main/java/io/ably/lib/transport/Defaults.java

```diff
@@ -2,9 +2,12 @@
 
 import io.ably.lib.types.ClientOptions;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class Defaults {
 	public static final int protocolVersion     = 1;
-	public static final String[] FALLBACK_HOSTS = new String[] {"A.ably-realtime.com", "B.ably-realtime.com", "C.ably-realtime.com", "D.ably-realtime.com", "E.ably-realtime.com"};
+	public static final List<String> FALLBACK_HOSTS = Arrays.asList("A.ably-realtime.com", "B.ably-realtime.com", "C.ably-realtime.com", "D.ably-realtime.com", "E.ably-realtime.com");
```

Comment:
List gives us useful apis without sacrificing performance considering we just have 5 items.